roxctl/deployment/check/check.go

```diff
@@ -115,6 +115,7 @@ func Command(cliEnvironment environment.Environment) *cobra.Command {
 	c.Flags().BoolVar(&deploymentCheckCmd.printAllViolations, "print-all-violations", false, "whether to print all violations per alert or truncate violations for readability")
 	c.Flags().BoolVar(&deploymentCheckCmd.force, "force", false, "bypass Central's cache for images and force a new pull from the Scanner")
 	utils.Must(c.MarkFlagRequired("file"))
+	c.Flags().StringVarP(&deploymentCheckCmd.cluster, "cluster", "", "", "cluster name or ID to delegate image scans to")
```

Comment:
Since we leave the short-hand empty (and I'd say `-c` is a bit weird), let's use [StringVar](https://pkg.go.dev/github.com/spf13/pflag#StringVar) instead.

On the description: we've had some discussion about specifying the cluster not only for the reason of delegating image scans but also e.g. in the context of policy evaluations. While it's currently not in place, let's adjust this and say "cluster name or ID to use as context for evaluation"?

Reply:
https://github.com/stackrox/stackrox/pull/7758/commits/4526051b51195713b6bb99118bf582415a993046